ccu3x86/hm_email-addon.cfg

```diff
@@ -1,9 +1,9 @@
 {
   CONFIG_URL /addons/email
   CONFIG_DESCRIPTION {
-		de {<li>Email-Integration</li><li>Alarmierung und Weiterleitung von Systemzustaenden</li><li>Einfache Benachrichtigungen</li><li>...</li>}
+		de {<li>E-Mail-Integration</li><li>Alarmierung und Weiterleitung von Systemzustaenden</li><li>Einfache Benachrichtigungen</li><li>...</li>}
 		en {<li>Email-Integration</li><li>Alerting and routing of System-States</li><li>Simple Notifications</li><li>...</li>}
   }
   ID email
-  CONFIG_NAME "Email"
+  CONFIG_NAME "E-Mail"
```

Comment:
Ich denke die Anpassung des CONFIG_NAME sollte hier vermieden werden weil ansonsten ggf. 2 einträge in der Systemsteuerung entstehen werden weil der alte eintrag mit ID Email ja weiter existeren wird.